lms/djangoapps/instructor_task/management/commands/fail_old_queueing_tasks.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import unicode_literals, print_function
+
+from datetime import timedelta
+
+from celery.states import FAILURE
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from lms.djangoapps.instructor_task.models import InstructorTask, QUEUING
+
+
+class Command(BaseCommand):
+    """
+    Command to manually fail old "QUEUING" tasks in the instructor task table.
+
+    Example:
+    ./manage.py lms fail_old_queueing_tasks --dry-run --days_previous 2 --task_type bulk_course_email
+    """
+
+    def add_arguments(self, parser):
+        """
+        Add arguments to the command parser.
+        """
+        parser.add_argument(
+            '--days-previous',
+            type=int,
+            dest='days_previous',
```

Comment:
Yeah, the idea is that it will fail all queueing tasks that were created before or up to the date represented here. But I think a window actually makes more sense. I'll update the PR